resources/views/register.blade.php

```diff
@@ -1,50 +1,222 @@
                                 </div>
                             </div>
                         </div>
                         <div class="row">
                             <div class="form-group col-md-12">
-                                <label class="col-md-3 control-lable" for="lastName">@lang('register.fullname')</label>
+                                <label class="col-md-3 control-lable" for="name">@lang('register.fullname')</label>
                                 <div class="col-md-9">
-                                    <input type="text" path="lastName" id="lastName" class="form-control input-sm" />
+                                    <input type="text" name="name" id="name" class="form-control input-sm" />
+                                    @error('name')
+                                        <span class="invalid-feedback" role="alert">
+                                            <span class="span-error">{{ $message }}</span>
+                                        </span>
+                                    @enderror
                                 </div>
                             </div>
                         </div>
                         <div class="row">
                             <div class="form-group col-md-12">
-                                <label class="col-md-3 control-lable" for="lastName">@lang('register.password')</label>
+                                <label class="col-md-3 control-lable" for="password">@lang('register.password')</label>
                                 <div class="col-md-9">
-                                    <input type="text" path="lastName" id="lastName" class="form-control input-sm" />
+                                    <input type="password" name="password" id="password" class="form-control input-sm" />
+                                    @error('password')
+                                        <span class="invalid-feedback" role="alert">
+                                            <span class="span-error">{{ $message }}</span>
+                                        </span>
+                                    @enderror
                                 </div>
                             </div>
                         </div>
                         <div class="row">
                             <div class="form-group col-md-12">
-                                <label class="col-md-3 control-lable" for="lastName">@lang('register.confirm-password')</label>
+                                <label class="col-md-3 control-lable"
+                                    for="password_confirmation">@lang('register.confirm-password')</label>
                                 <div class="col-md-9">
-                                    <input type="text" path="lastName" id="lastName" class="form-control input-sm" />
+                                    <input type="password" name="password_confirmation" id="password_confirmation"
```

Comment:
```suggestion
                                    <input type="password" name="password_confirmation" id="password-confirmation"
```
id, class thì nên viết kiểu này